samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -38,6 +38,14 @@ Configuration
 
 |config|
 
+This sample configuration is split into two parts:
+* generic configuration is available in :file:`prj.conf` file
+* board specific configuration is avaialbe in :file:`boards/<BOARD>.conf` file
```

Comment:
```suggestion
This sample configuration is split into the following files:

* generic configuration is available in :file:`prj.conf` file
* board specific configuration is available in :file:`boards/<BOARD>.conf` file
```

Reply:
Changed: added empty line and changed first sentence.